peachjam/views/legislation.py

```diff
@@ -156,19 +186,25 @@ def get_timeline_events(self):
                 }
             )
 
-        amendments = self.object.metadata_json.get("work_amendments", None)
-        if amendments:
-            events.extend(
-                [
-                    {
-                        "date": amendment.get("date"),
-                        "event": "amendment",
-                        "amending_title": amendment.get("amending_title"),
-                        "amending_uri": amendment.get("amending_uri"),
-                    }
-                    for amendment in amendments
-                ]
-            )
+        amendments = self.get_work_amendments()
+        if points_in_time and amendments:
+            point_in_time_dates = [
+                point_in_time["date"] for point_in_time in points_in_time
+            ]
+            event = [
+                {
+                    "date": amendment.get("date"),
+                    "event": "amendment",
+                    "amending_title": amendment.get("amending_title"),
+                    "amending_uri": amendment.get("amending_uri"),
+                    "flag": True
```

Comment:
I think it looks good and seems to be doing the right thing, I would just query naming the flag "flag". maybe something more descriptive like `unapplied_amendment` will help us in future?

Reply:
Good call